src/Presentation/ExportConfigEditPageTextBuilder.php

```diff
@@ -17,144 +19,210 @@ public function __construct(
-			'<th>' . $this->context->msg( 'wikibase-export-config-help-table-example' )->escaped() . '</th>' .
-			'</tr></thead>' .
-			'<tbody>' .
-			$this->createTableRow(
-				'startTimePropertyId',
-				'wikibase-export-config-help-variable-start-time-property-id',
-				'"P100"',
-			) .
-			$this->createTableRow(
-				'endTimePropertyId',
-				'wikibase-export-config-help-variable-end-time-property-id',
-				'"P200"',
-			) .
-			$this->createTableRow(
-				'pointInTimePropertyId',
-				'wikibase-export-config-help-variable-point-in-time-property-id',
-				'"P300"',
-			) .
-			$this->createTableRow(
-				'propertiesToGroupByYear',
-				'wikibase-export-config-help-variable-properties-with-qualifiers',
-				'[ "P1", "P2" ]',
-			) .
-			$this->createTableRow(
-				'ungroupedProperties',
-				'wikibase-export-config-help-variable-properties-without-qualifiers',
-				'[ "P3", "P4" ]',
-			) .
-			$this->createTableRow(
-				'defaultSubjects',
-				'wikibase-export-config-help-variable-default-subjects',
-				'[ "Q1", "Q2" ]',
-			) .
-			$this->createTableRow(
-				'defaultStartYear',
-				'wikibase-export-config-help-variable-default-start-year',
-				'2010',
-			) .
-			$this->createTableRow(
-				'defaultEndYear',
-				'wikibase-export-config-help-variable-default-end-year',
-				'2022',
-			) .
-			$this->createTableRow(
-				'subjectFilterPropertyId',
-				'wikibase-export-config-help-variable-subject-filter-property-id',
-				'"P50"',
-			) .
-			$this->createTableRow(
-				'subjectFilterPropertyValue',
-				'wikibase-export-config-help-variable-subject-filter-property-value',
-				'"company"',
-			) .
-			$this->createTableRow(
-				'exportLanguages',
-				'wikibase-export-config-help-variable-export-languages',
-				'[ "en", "nl" ]',
-			) .
-			'</tbody>' .
-			'</table>';
+	public function createBottomHtml(): string {
+		return <<<HTML
+<div id="Documentation">
+	<section>
+		<h2 id="ConfigurationDocumentation">{$this->context->msg( 'wikibase-export-config-help' )->escaped()}</h2>
+
+		<p>
+			Besides the configuration reference below, you can consult the Wikibase Export
+			<a href="https://professional.wiki/en/extension/wikibase-export">usage documentation</a> and
+			<a href="https://export.wikibase.wiki">demo wiki</a>.
+		</p>
+	</section>
+
+	<section>
+		<h2 id="ExportLanguage">Export language</h2>
+
+		<p>
+			By default the export happens in the wikis main language.
+		</p>
+
+		<p>
+			You can change the export language or let the user choose by specifying multiple available languages.
+		</p>
+
+		<p>
+			Example configuration:
+		</p>
+
+		<pre>
+{
+	"exportLanguages": [
+		"en",
+		"nl",
+		"de"
+	]
+}</pre>
+	</section>
+
+	<section>
+		<h2 id="ExportSubjects">Export subjects</h2>
+
+		<p>
+			You can specify the subjects that should be selected for export by default.
+		</p>
+
+		<pre>
+{
+    "defaultSubjects": [
+        "Q1",
```

Comment:
Some of the JSON snippets use spaces and others use tabs.
![Screenshot_20230205_113940](https://user-images.githubusercontent.com/1428594/216811824-51da86cb-a075-4a6e-a7d4-ff3f3b417541.png)

The tab snippets look a bit wide, although it's probably better for copy-paste purposes since our default JSON in the textarea uses tabs.